packages/adapters/gupshup-whatsapp/src/GupShupWhatsappAdapter.ts

```
@@ -299,15 +302,16 @@
-    xmsg = this.manageConversation(xmsg)
+    xmsg = this.manageConversation(xmsg);
     return xmsg;
   }
 
   private manageConversation(xmsg: XMessage): XMessage {
+    console.log("===================")
     const currentTimestamp = Date.now();
     const lastMessageTimestamp = this.getLastMessageTimestamp(this.userHistory);
-
-    if (this.shouldCreateNewConversation(lastMessageTimestamp, currentTimestamp)) {
+    console.log(`execution result: ${this.callback && this.callback(this.userHistory)}`)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)